### ISSUE TITLE
Fails with "SSH isn't starting" with SSH option "Compression yes"

### ISSUE THREAD

**Issue by svend** [datawire/telepresence#97]
### What were you trying to do?

Start a telepresence shell. I have the SSH option "Compression yes" set in `~/.ssh/config`.

### What did you expect to happen?

The shell would start.

### What happened instead?

There is a long pause after "Starting proxy...", then the traceback below is printed. If I removed the Compression setting from my SSH config, the command is successful.

### Automatically included information

Command line: `['/Users/ssorensen/bin/telepresence', '--new-deployment', 'quickstart', '--run-shell']`
Version: `0.26`
Python version: `3.6.0 (default, Mar 27 2017, 20:59:04) 
[GCC 4.2.1 Compatible Clang 3.7.1 (tags/RELEASE_371/final)]`
kubectl version: `Client Version: v1.5.3-dirty`
OS: `Darwin 200956 16.5.0 Darwin Kernel Version 16.5.0: Fri Mar  3 16:52:33 PST 2017; root:xnu-3789.51.2~3/RELEASE_X86_64 x86_64 i386 MacBookPro12,1 Darwin`
Traceback:

```
Traceback (most recent call last):
  File "/Users/ssorensen/bin/telepresence", line 678, in call_f
    return f(*args, **kwargs)
  File "/Users/ssorensen/bin/telepresence", line 761, in go
    subprocesses, env, socks_port = start_proxy(runner, args)
  File "/Users/ssorensen/bin/telepresence", line 507, in start_proxy
    args.expose,
  File "/Users/ssorensen/bin/telepresence", line 433, in connect
    wait_for_ssh(runner, ssh_port)
  File "/Users/ssorensen/bin/telepresence", line 379, in wait_for_ssh
    raise RuntimeError("SSH isn't starting.")
RuntimeError: SSH isn't starting.

```

Logs:

```
le=/dev/null', 'root@localhost', '/bin/true'],)
Running: (['ssh', '-q', '-p', '60325', '-oStrictHostKeyChecking=no', '-oUserKnownHostsFile=/dev/null', 'root@localhost', '/bin/true'],)
Running: (['ssh', '-q', '-p', '60325', '-oStrictHostKeyChecking=no', '-oUserKnownHostsFile=/dev/null', 'root@localhost', '/bin/true'],)
Running: (['ssh', '-q', '-p', '60325', '-oStrictHostKeyChecking=no', '-oUserKnownHostsFile=/dev/null', 'root@localhost', '/bin/true'],)
Running: (['ssh', '-q', '-p', '60325', '-oStrictHostKeyChecking=no', '-oUserKnownHostsFile=/dev/null', 'root@localhost', '/bin/true'],)
Running: (['ssh', '-q', '-p', '60325', '-oStrictHostKeyChecking=no', '-oUserKnownHostsFile=/dev/null', 'root@localhost', '/bin/true'],)
Running: (['ssh', '-q', '-p', '60325', '-oStrictHostKeyChecking=no', '-oUserKnownHostsFile=/dev/null', 'root@localhost', '/bin/true'],)
Running: (['ssh', '-q', '-p', '60325', '-oStrictHostKeyChecking=no', '-oUserKnownHostsFile=/dev/null', 'root@localhost', '/bin/true'],)

```

**Comment by itamarst** [comment]
Thanks for the bug report!

Is compression the only custom option you have, or was there some other hint that led you test that as a possible cause?

**Comment by itamarst** [comment]
Was able to reproduce this locally. My guess this is due to https://bugs.alpinelinux.org/issues/6651 - solutions are either disabling compression server-side (I verified that works) or switch to newer Alpine image. I will try the latter, falling back to former if that doesn't help.

**Comment by itamarst** [comment]
Newer Alpine appears to fix the problem.

**Comment by itamarst** [comment]
This has been released in Telepresence 0.27. Please let us know if the problem is still happening.

**Comment by svend** [comment]
Upgrading to 0.27 fixed the issue for me. Thanks for tracking this down!
